src/lib/components/composites/input/ChipsInput.svelte

```diff
@@ -0,0 +1,214 @@
+<script lang="ts">
+    import { Label } from "$lib/components/primitives/label";
+    import { Button } from "$lib/components/primitives/button";
+
+    interface ChipsInputProps {
+        items: string[];
+        label?: string;
+        labelPosition?: "top" | "left";
+        searchSuggestions?: (arg0: string) => string[];
+        placeholder?: string;
+    }
+
+    let {
+        items = $bindable(),
+        label,
+        labelPosition = "top",
+        searchSuggestions,
+        placeholder = "",
+    }: ChipsInputProps = $props();
+
+    /*
```

Comment:
Add another asterisk so that it counts as documentation for `selectedChipIndex`
```suggestion
    /**
```

Reply:
Also, \<length of items\> isn't displayed in VS Code. I recommend using square brackets.

Does it display the comment correctly in WebStorm?

Reply:
For both comments, it's the same for `selectedSuggestionIndex`

Reply:
I will change that, didn't notice it, because Webstorm is smart enough to completely show the comment.